packages/edit-navigation/src/components/menu-editor/block-editor-panel.js

```diff
@@ -50,34 +52,40 @@ export default function BlockEditorPanel( {
 	return (
 		<Panel className="edit-navigation-menu-editor__block-editor-panel">
 			<PanelBody title={ __( 'Navigation menu' ) }>
-				<div className="components-panel__header-actions">
-					<Button isPrimary onClick={ saveBlocks }>
-						{ __( 'Save navigation' ) }
-					</Button>
-				</div>
-				<NavigableToolbar
-					className={ classnames(
-						'edit-navigation-menu-editor__block-editor-toolbar',
-						{
-							'is-hidden': isNavigationModeActive,
-						}
-					) }
-					aria-label={ __( 'Block tools' ) }
-				>
-					{ hasSelectedBlock && <BlockToolbar hideDragHandle /> }
-				</NavigableToolbar>
-				<Popover.Slot name="block-toolbar" />
-				<WritingFlow>
-					<ObserveTyping>
-						<BlockList />
-					</ObserveTyping>
-				</WritingFlow>
-				<div className="components-panel__footer-actions">
-					<DeleteMenuButton
-						menuId={ menuId }
-						onDelete={ onDeleteMenu }
-					/>
-				</div>
+				{ hasMenu && (
```

Comment:
Should this component just return null when `hasMenu === false`?

Reply:
I ended up removing that and going with the suggestion of @paaljoachim below and showing the create panel alongside other panels.